assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -545,12 +545,12 @@ che.factory.default_plugins=eclipse/che-machine-exec-plugin/nightly
 # Default Editor that should be provisioned into Devfile if there is no specified Editor
 # Format is `editorPublisher/editorName/editorVersion` value.
 # `NULL` or absence of value means that default editor should not be provisioned.
-che.workspace.devfile.default_editor=eclipse/che-theia/next
+che.workspace.devfile.default_editor=quay.io/eclipse/che-theia/next
```

Comment:
Not sure about this change. This is id of an editor, not the image id.

Reply:
you're totally right. 
Images are defined in Plugin Registry and it's already changed to quay.io in https://github.com/eclipse/che-plugin-registry/pull/323